WTM_SWITCHER

```diff
@@ -54,11 +54,25 @@ def saveTopCoin(data):
     logFile.close()
     return
 
-#Get BTC exchange rate from blockchain.info
+#Get BTC exchange rate 
+try:
+    exchrate=float(json.loads(requests.get("http://api.coindesk.com/v1/bpi/currentprice.json").read())['bpi'][currency]['rate_float'])
+    print exchrate
+except:
+    print("Can not get data from coindesk.com")
+    print("Trying to get data from coinbase.com")
+try:
```

Comment:
You have to indent this (and others below) to be nested. Alternatively, move that code section to a function and insert a `return` instruction after each `print exchrate`. Move the "Tying…" message inside the try block instead of leaving in the exception.

Reply:
Can you please do it, you know I have no knowledge in python